src/Commands/core/UpdateDBCommands.php

```diff
@@ -672,7 +673,12 @@ public function updateCheckRequirements()
                 if (isset($requirement['severity']) && $requirement['severity'] != REQUIREMENT_OK) {
                     $message = isset($requirement['description']) ? $requirement['description'] : '';
```

Comment:
Probably we need to pass 'description' as well?

Reply:
Sure, lets do that

Reply:
@weitzman already did :)